metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/monitor/RegistryMonitor.java

```diff
@@ -51,16 +48,14 @@ public ProducerInvocationMonitor getProducerInvocationMonitor(String operationNa
     return producerInvocationMonitors.computeIfAbsent(operationName, i -> new ProducerInvocationMonitor(operationName));
   }
 
-  public RegistryMetric toRegistryMetric(int windowTimeIndex) {
-    Map<String, ConsumerInvocationMetric> consumerInvocationMetrics = new HashMap<>();
+  public Map<String, Double> measure(int windowTimeIndex, boolean calculateLatency) {
+    Map<String, Double> measurements = new HashMap<>(systemMonitor.measure());
```

Comment:
If we want measure the invocation monitor, we don't need to know if it consumerInvocationMonitor or producerInvocationMonitor.

Reply:
Done